src/iptc.cpp

```diff
@@ -79,6 +80,90 @@ namespace {
         uint16_t record_;
 
     }; // class FindIptcdatum
+
+    bool is_utf8(const std::string& string)
+    {
+        if(string.empty())
+            return 0;
```

Comment:
Could you change the `return 0/1` to `return false/true`?

Reply:
Sure, I'll have that into account for the next iteration ;)